docker-compose.yml

```diff
@@ -13,6 +13,8 @@ services:
     volumes:
       - ./:/chroma
       - index_data:/index_data
+      - /etc/localtime:/etc/localtime:ro
```

Comment:
I am wondering if using TZ env var has also some merit here.

Some considerations when mounting volumes:
- Portability - this will not work on some windows setups, possibly just fine for all Linux and MacOS distros
- If one wants to run the container in a specific TZ different than the hosts then different mounts are needed.
- Security of mounting host file systems (granted that localtime and timezone is generally safe)

Reply:
I think we should consider two options:
- Volumes (good for testing and dev)
- TZ env var, more universal but requires tzdata package and of course instructions for users how to create/set the env var.

Ideally, we should be mindful of the developer experience on local machines as well as cloud resources (including K8s)

here are some examples how to get the local TZ on various OS:

Mac - `TZ=$(systemsetup -gettimezone | cut -d ' ' -f 2-)`
Linux - `TZ=$(cat /etc/timezone)`
Windows - `$tz = (Get-TimeZone).Id` (I have not check this though as I almost never use Windows machines)